app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/directory/ContactDetailsFragmentTest.kt

```diff
@@ -76,11 +75,29 @@ class ContactDetailsFragmentTest {
     }
 
     @Test
-    fun deleteButtonIsDisplayed() {
+    fun deleteButtonIsDisplayedABoss() {
```

Comment:
You should change to either ...AsBoss or ...ForABoss.

Reply:
Yes my bad forgot the For